### PR TITLE
Add BlocksHelper class

### PR DESCRIPTION
**Pull request by kbeker**
No issue for that. Related with: https://github.com/golemfactory/golem-smart-contracts-interface/pull/51

### CODE REVIEW

File: concent_api/core/utils.py [hunk 1]
```diff
@@ -176,3 +178,25 @@ def adjust_format_name(output_format: str) -> str:
     if output_format.upper() == 'JPG':
         return 'JPEG'
     return output_format.upper()
+
+
+class BlocksHelper:
+    def __init__(self, sci: SmartContractsInterface) -> None:
+        self._sci = sci
+
+    def get_latest_existing_block_at(self, timestamp: int) -> Block:
```
Comment (Jakub89) [inline concent_api/core/utils.py]: That `at` at the end of this name is confusing. Can't this function be called `get_latest_existing_block()`?

Comment (kbeker) [reply concent_api/core/utils.py]: @Jakub89 maybe `get_latest_existing_block_since`?

Comment (dybi) [reply concent_api/core/utils.py]: @Jakub89 @kbeker I am fine with `get_latest_existing_block_at`

File: concent_api/core/tests/test_utils.py [hunk 2]
```diff
@@ -215,3 +217,27 @@ def test_that_method_returns_correct_format_name(output_format, expected):
...
+
+
+class BlocksHelperTest(TestCase):
+    def setUp(self):
+        self.sci = mock.Mock()
+        self.blocks_helper = BlocksHelper(self.sci)
+
+    def test_get_latest_existing_block_at(self):
+        blocks = []
+        for i in range(9):
```
Comment (dybi) [inline concent_api/core/tests/test_utils.py]: this is a part of  test's setup as well: please either move it to `setUp` method or get rid of that method and set everything up here.

Comment (kbeker) [reply concent_api/core/tests/test_utils.py]: Done. I just moved it to `setUp`

File: concent_api/core/tests/test_utils.py [hunk 3]
```diff
@@ -215,3 +217,51 @@ def test_that_method_returns_correct_format_name(output_format, expected):
...
+            5: 90,
+            6: 105,
+            7: 120,
+            8: 135,
+            9: 150,
+        }
+        self.sci = mock.Mock()
+        self.blocks_helper = BlocksHelper(self.sci)
+        self.sci.get_block_number.return_value = 9
+        self.sci.get_block_by_number.side_effect = lambda number: self.mocked_get_block_by_number(number)
```
[the remark this answers is not in the document]
Comment (kbeker) [reply concent_api/core/tests/test_utils.py]: Ou, right. Done

File: concent_api/core/tests/test_utils.py [hunk 4]
```diff
@@ -215,3 +217,51 @@ def test_that_method_returns_correct_format_name(output_format, expected):
...
+        (15, 1),
+        (31, 2),
+        (50, 3),
+        (74, 4),
+        (150, 9),
+        (151, 9),
+        (71830, 9),
+    ])
+    def test_get_latest_existing_block_at(self, timestamp, block_number):
+        assert_that(
```
[the remark this answers is not in the document]
Comment (kbeker) [reply concent_api/core/tests/test_utils.py]: Done

**Pull request merged**